pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -170,12 +171,41 @@ func (br *OVSBridge) create() Error {
 		Row:   bridge,
 	})
 
-	mutateSet := helpers.MakeOVSDBSet(map[string]interface{}{
+	interf := Interface{
```

Comment:
Questions:
1. Do we need to explicitly create the bridge interface? I thought even we not explicitly create it, an interface will always be created for the bridge, no?
2. Should we create the interface for Linux?
@wenyingd

Reply:
We need to update the comments on the public func to mention the bridge interface creation.

Reply:
No, if we don't create interface explicitly, the OVS port/interface is not created by default. It is a different behavior from ovsdb_client to command line "ovs-vsctl". If we create a bridge with command line, the OVS implementations of "ovs-vsctl" creates a port/interface with the same name as the bridge automatically. But with ovsdb_client, since we only define a table record, the port/interface table has no records added.

Rui and I have double checked Antrea on Linux, no "br-int" interface/port is created.

I don't think Linux need such interface according to current implementation. Since Antrea on Linux doesn't need the ovs bridge to help forward packet, we don't need such a interface. For Windows, this interface is used to take over the network configuration which was on the uplink interface.

Reply:
Ok. Then for Linux should we skip the interface creation?

Reply:
Yes, I think this step should be called only on Windows.

Reply:
Thanks Jianjun. I move the "add local port" logic from create() to agent_windows.prepareOVSBridge()